flutter/lib/jni/jni_raw.dart

```diff
@@ -82,25 +88,38 @@ class JniObject extends NativeFieldWrapperClass2 {
       native 'JniObject_CallVoidMethod';
 }
 
-/// Wrapper for a Java class accessed via JNI.
+/// Low-level wrapper for a Java class accessed via JNI.
 class JniClass extends JniObject {
+  /// Loads the Java class with the given fully qualified name.
   static JniClass fromName(String name)
       native 'JniClass_FromName';
+
+  /// Returns a wrapper for a `java.lang.Class` object.
   static JniClass fromClassObject(JniObject classObject)
       native 'JniClass_FromClassObject';
 
+  /// Returns a field ID for the nonstatic field matching this name and type signature.
+  /// See the JNI reference for explanation of type signatures.
   int getFieldId(String name, String sig)
       native 'JniClass_GetFieldId';
+
+  /// Returns a field ID for a static field.
```

Comment:
the descriptions of the methods in this class need more colour. right now they basically just say the same thing as the method name.